File: lecture_003(Pattern)/l_013_pattern13.java
import java.util.Scanner;
public class l_013_pattern13 {
    public static Scanner scn = new Scanner(System.in);
    public static void main(String[] args) {
        int n = scn.nextInt();
        //rajneeshPattern9(n);
        sumeetPattern9(n);
    }

    /*public static void rajneeshPattern9(int n) {
        int nst = n;
        for(int r=1; r<=n; r++) {
            for(int cst=1; cst<=nst; cst++) {
                if(r==cst || r+cst==n+1) {
                    System.out.print("*");
                }
                else 
                    System.out.print(" ");
            }
            System.out.println();
        }
    }*/

    public static void sumeetPattern9(int n) {
        for(int r=1; r<=n; r++) {
            for(int cst=1; cst<=n; cst++) {
                if(r==cst || r+cst==n+1) {
                    System.out.print("*");
                }
                else
                    System.out.print(" "); 
            }
            System.out.println();
        }
    }
}

File: lecture_003(Pattern)/l_013_pattern13.cpp
#include<iostream>
using namespace std;

/*void rajneeshpattern9(int n) {
    int nst = 1;
    int nsp = n-1;
    for(int r=1; r<=n; r++) {
        for(int csp=1; csp<=nsp; csp++) {
            cout<<" ";
        }
        for(int cst=1; cst<=nst; cst++) {
            cout<<"*";
        }
        nsp--;
        cout<<endl;
    }
}*/

void sumeetpattern9(int n) {
    for(int r=1; r<=n; r++) {
        for(int cst=1; cst<=n; cst++) {
            if(r==cst || r+cst==n+1) {
                cout<<"*";
            }
            else
                cout<<" "; 
        }
        cout<<endl;
    }
}
int main(int args, char**argv) {
    int n;
    cin>>n;
    //rajneeshpattern9(n);
    sumeetpattern9(n);
    return 0;
}